server/config/passport.js

```diff
@@ -0,0 +1,24 @@
+const JwtStrategy = require("passport-jwt").Strategy;
+const ExtractJwt = require("passport-jwt").ExtractJwt;
+const mongoose = require("mongoose");
+const User = mongoose.model("users");
+const keys = require("../config/keys");
+
+const opts = {};
+opts.jwtFromRequest = ExtractJwt.fromAuthHeaderAsBearerToken();
+opts.secretOrKey = keys.secretOrKey;
+
+module.exports = passport => {
+	passport.use(
+		new JwtStrategy(opts, (jwt_payload, done) => {
+			User.findById(jwt_payload.id)
```

Comment:
I believe you are actually not saving your id in the JWT payload, but the email? Either choose to save id (which is standard and more efficient), or you need to fetch user by email here